app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdScreen.kt

```diff
@@ -360,6 +362,9 @@ class ScanSimpleIdScreen : BaseScreen<
   interface Injector {
     fun inject(target: ScanSimpleIdScreen)
   }
+
+  @Parcelize
+  data class SendScannedIdentifier(val identifier: Identifier) : Parcelable
```

Comment:
I think we can just rename this to `ScannedIdentifier`

Reply:
ScannedIdentifier does not define the variable really well, hence the name

Reply:
No, I meant, we don't need to add the `Send` prefix. This type is only referred to as the `Succeeded` result. And it doesn't actually refer to us sending anything.

Reply:
It is sending result, maybe you can look at `SendBlankScannedQrCodeResult` to get the better idea of what's happening?

Reply:
That's an effect name. The request type it's passing is also named as `BlankScannedQRCodeResult`

Reply:
> The idea is to make the code readable and to understand what's happening better

That is correct but in this case, this data class doesn't refer to an act of sending something but it just refers to the `ScannedQrCodeResult` or `ScannedQRCode` the action `router.popWithResult` is actually returning this back.